tools/editor/plugins/script_editor_plugin.cpp

```diff
@@ -2743,7 +2768,8 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 	file_menu->get_popup()->add_shortcut(ED_SHORTCUT("script_editor/save_theme", TTR("Save Theme")), FILE_SAVE_THEME);
 	file_menu->get_popup()->add_shortcut(ED_SHORTCUT("script_editor/save_theme_as", TTR("Save Theme As")), FILE_SAVE_THEME_AS);
 	file_menu->get_popup()->add_separator();
-	file_menu->get_popup()->add_shortcut(ED_SHORTCUT("script_editor/close_file", TTR("Close"), KEY_MASK_CMD|KEY_W), FILE_CLOSE);
+	file_menu->get_popup()->add_shortcut(ED_SHORTCUT("script_editor/close_docs", TTR("Close Docs"), KEY_MASK_CMD | KEY_D), CLOSE_DOCS);
```

Comment:
I would leave it without any default hotkey. For such special cases, it's not a problem to go to the menu. From 2.1 you can also change the hotkey if the option has a ShortCut.